frontend/crd/src/app/dashboard/events/events.component.html

```diff
@@ -4,15 +4,18 @@ <h2>
       Events
     </h2>
   </mat-card-header>
-  <mat-card-content>
-    <ng-container *ngFor="let event of events">
-      <div>{{event.name}}</div>
-      <div>{{event.description}}</div>
-      <div>{{event.isRequired? "Required": "Optional"}}</div>
-      <div>{{event.date}}</div>
-      <div>{{event.organizer}}</div>
-      <div>{{event.location}}</div>
-      <div>&nbsp;</div>
-    </ng-container>
-  </mat-card-content>
+  <carousel [itemsPerSlide]="itemsPerSlide"
+            [singleSlideOffset]="true"
+            [noWrap]="!noWrap"
+            [interval]="0"
+            [startFromIndex]="0">
+    <slide *ngFor="let slide of slides; let index=index">
+      <img [src]="slide.img" alt="event image" style="display: block; width: 100%; padding: 1%; background-color: darkslategrey;">
```

Comment:
You should do this with the Oswego logo component. just add an input and set this styling conditionally in the less

Reply:
set the background image as the oswego logo component?

Reply:
leaving this as a placeholder